### PR TITLE
Bug Fixes in Website Directory

### PR DESCRIPTION
**Pull request by brookescarlett**
- Remove positive look ahead regex bc Safari browser does not have support
- Combine use of Text and Area knobs in LiveExamples
- Add more descriptive meta tags to Head

### CODE REVIEW

File: website/components/TypeDefinition.tsx [hunk 1]
```diff
@@ -14,7 +15,11 @@ function TypeDefinition({ markdownAst, readme }) {
     .map(treeItem => treeItem.children?.[0].value);
 
   // Gets types defined in readmes to expand upon below the prop table
-  const interfaceDefinitions = readme?.match(/(?<=typescript).*?(?=```)/gs);
+  const getTypes: Array<string> = readme?.split('typescript');
```
Comment (lazytype) [inline website/components/TypeDefinition.tsx:18]:
```suggestion
  const getTypes: Array<string> = readme.split('typescript');
```

Also shouldn't need the explicit type if you add a type for the props

File: website/components/TypeDefinition.tsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { css } from 'emotion';
 import Code from '@leafygreen-ui/code';
 import { InlineCode } from '@leafygreen-ui/typography';
+import { spacing } from '@leafygreen-ui/tokens';
 import { readmeDepthMap, HeadingType } from './PropTable';
 
 function TypeDefinition({ markdownAst, readme }) {
```
Comment (lazytype) [inline website/components/TypeDefinition.tsx:8]: this is lacking types. should probably figure out what's wrong with the TS configuration

File: website/components/TypeDefinition.tsx [hunk 3]
```diff
@@ -14,7 +15,11 @@ function TypeDefinition({ markdownAst, readme }) {
     .map(treeItem => treeItem.children?.[0].value);
 
   // Gets types defined in readmes to expand upon below the prop table
-  const interfaceDefinitions = readme?.match(/(?<=typescript).*?(?=```)/gs);
+  const getTypes: Array<string> = readme?.split('typescript');
+  getTypes.shift();
+  const interfaceDefinitions = getTypes?.map(id => {
+    return id.split('```')[0];
+  });
```
Comment (lazytype) [inline website/components/TypeDefinition.tsx:22]:
```suggestion
  getTypes.shift();

  const interfaceDefinitions = getTypes.map(id => id.split('```')[0]);
```

File: website/components/live-example/Knobs.tsx [hunk 4]
```diff
@@ -125,6 +126,31 @@ function Number({ onChange, label, value, prop, darkMode }: NumberInterface) {
 }
 
 function Text({ onChange, label, value, prop, darkMode }: TextInterface) {
+  const handleChange = ({ target }: React.ChangeEvent<HTMLInputElement>) => {
```
Comment (lazytype) [inline website/components/live-example/Knobs.tsx:129]: `useCallback`

File: website/components/live-example/Knobs.tsx [hunk 5]
```diff
@@ -125,6 +126,31 @@ function Number({ onChange, label, value, prop, darkMode }: NumberInterface) {
...
+    <div className={knobContainerStyle}>
+      <label
+        className={cx(labelStyle, { [labelDarkMode]: darkMode })}
+        id={labelId}
+      >
+        {label}
+      </label>
+      <TextInput
+        onChange={handleChange}
+        value={value.toString()}
```
Comment (lazytype) [inline website/components/live-example/Knobs.tsx:145]: `value` is already a string

**Review by lazytype — CHANGES_REQUESTED** [review]

File: website/components/TypeDefinition.tsx (hunk 1 above)
Comment (brookescarlett) [reply website/components/TypeDefinition.tsx:18]: Yep got it

File: website/components/TypeDefinition.tsx (hunk 2 above)
Comment (brookescarlett) [reply website/components/TypeDefinition.tsx:8]: fixed, but still confused re TS configuration. I thought I already solved for this 🤔

File: website/components/TypeDefinition.tsx (hunk 3 above)
Comment (brookescarlett) [reply website/components/TypeDefinition.tsx:22]: 👍

File: website/components/live-example/Knobs.tsx (hunk 4 above)
Comment (brookescarlett) [reply website/components/live-example/Knobs.tsx:129]: 👍

File: website/components/live-example/Knobs.tsx (hunk 5 above)
Comment (brookescarlett) [reply website/components/live-example/Knobs.tsx]: 👍

File: website/components/TypeDefinition.tsx [hunk 6]
```diff
@@ -2,19 +2,33 @@ import React from 'react';
...
         treeItem.depth === readmeDepthMap[HeadingType.TypeDefinition],
     )
-    .map(treeItem => treeItem.children?.[0].value);
+    .map((treeItem: Heading) => treeItem.children?.[0].value);
 
   // Gets types defined in readmes to expand upon below the prop table
-  const interfaceDefinitions = readme?.match(/(?<=typescript).*?(?=```)/gs);
+  const getTypes = readme.split('typescript');
+  getTypes.shift();
+  const interfaceDefinitions = getTypes?.map(id => id.split('```')[0]);
```
Comment (lazytype) [inline website/components/TypeDefinition.tsx:31]:
```suggestion
  const interfaceDefinitions = getTypes.map(id => id.split('```')[0]);
```

**Review by lazytype — APPROVED** [review]

**Pull request merged**